### PR TITLE
🎬 Workflow Actions

### LINKED ISSUE #74: CD workflow
Copy CD workflow from Chowda to auto build images with GitHub Actions

### PR DESCRIPTION
**Pull request by mrharpo**
# Actions
Adds actions from Chowda:
- CI
- CD
- Release
- Publish
- Update
- Docs

Closes #74

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ services:
     build:
       context: .
       target: base
-    image: ov-wag
+    image: ghcr.io/wgbh-mla/ov-wag:pr-76
```
Comment (afred) [inline docker-compose.yml:8]: Isn't `image` ignored in the presence of `build`? [From the docs](https://docs.docker.com/compose/compose-file/build/#definitions):
> When a build subsection is present for a service, Compose ignores the image attribute for the corresponding service, as Compose can build an image from source.

File: test.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    build:
+      context: .
+      target: test
+    image: ov-tests
+    volumes:
+      - ./:/app/
+    entrypoint: /app/docker_entrypoints/test.sh
+    environment:
+      - OV_DB_ENGINE=django.db.backends.sqlite3
+      - OV_DB_NAME=testdb
```
Comment (afred) [inline test.yml:14]: I can see where the top-level `dev.yml` is used in the `ov` script, but i'm not seeing where `dev.yml` is used.

File: docker-compose.yml (hunk 1 above)
Comment (mrharpo) [reply docker-compose.yml:8]: Yes, that's leftover from the bootstrapping process, from when my computer refused to build the image.

File: test.yml (hunk 2 above)
Comment (mrharpo) [reply test.yml:14]: Forthcoming: https://github.com/WGBH-MLA/ov-wag/pull/83/files#diff-fe4c6e543bb224b046d34d6c9c3f9600162e576dc2512c670e91998dff2511a1

Though, it is incorrectly referred to as `tests.yml`. Will fix it there.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
-      - name: 📰 Checkout
-        uses: actions/checkout@v3
+  tests:
+    name: ⚗️ Application Tests
+    uses: WGBH-MLA/.github/.github/workflows/pytest-with-postgres.yml@main
+    secrets: inherit
+    with:
+      pdm_args: -G test,ci
+      pytest_args: -n auto --nbmake -ra -s
+      pg_db: chowda
```
[the remark this answers is not in the document]
Comment (mrharpo) [reply .github/workflows/ci.yml:13]: 🤣

**Review by afred — APPROVED** [review]
👍

**Pull request merged**